web_requester.go

```diff
@@ -109,6 +110,15 @@ func (w *WebRequesterFactory) GetRequester(uint64) bench.Requester {
 		w.expandedHeaders = expandedHeaders
 	}
 
+	// if BodyFile is specified Body is ignored
+	if w.BodyFile != "" {
+		content, err := ioutil.ReadFile(w.BodyFile)
+		if err != nil {
```

Comment:
You don't need the `nil` check, `maybePanic` will do nothing if `err` is `nil`. Please remove.